File: Backend/app/src/main/java/com/example/demo/dto/UserDto.java
package com.example.demo.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class UserDto {
    private Long id;                 // nullable
    private String fullName;
    private String email;
    private String phoneNumber;
    private String username;
    private String imgUrl;          // nullable
    private String gender;
    private String dateOfBirth;
}

File: Mobile/app/src/main/java/com/example/myapplication/data/dto/UserDto.kt
package com.example.myapplication.data.dto

class UserDto (
    val id: Long? = null,
    val fullName: String?=null,
    val email: String?=null,
    val phoneNumber: String?=null,
    val username: String?=null,
    val imgUrl: String? = null,
    val gender: String?=null,
    val dateOfBirth: String?=null   // Format: yyyy-MM-dd (giống LocalDate)
)